src/routes/goalRoute.ts

```diff
@@ -1,20 +1,20 @@
-import { Router } from "express"
```

Comment:
Remove file. Work related to Goals will be done in separate branches per crud operation